ecms_base/features/custom/ecms_publications/config/install/field.field.node.publication.field_publication_file_download.yml

```diff
@@ -0,0 +1,27 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_publication_file_download
+    - media.type.publication_file
+    - node.type.publication
+id: node.publication.field_publication_file_download
+field_name: field_publication_file_download
+entity_type: node
+bundle: publication
+label: 'Publication File Download'
+description: 'Select a file from the media library'
+required: false
+translatable: false
+default_value: {  }
+default_value_callback: ''
+settings:
+  handler: 'default:media'
+  handler_settings:
+    target_bundles:
+      publication_file: publication_file
+    sort:
+      field: _none
+    auto_create: false
+    auto_create_bundle: event_image
```

Comment:
Why is there an `event_image` reference here?

Reply:
Well that was odd - could not get it to update, so had to manually import the config - which seems to have worked.